suite-common/wallet-core/src/send/sendFormSelectors.ts

```diff
@@ -73,3 +73,5 @@ export const selectSendFormReviewButtonRequestsCount = (
 
     return isCardano ? sendFormReviewRequest.length - 1 : sendFormReviewRequest.length;
 };
+
+export const selectAreFeesLoading = (state: SendRootState) => state.wallet.send.areFeesLoading;
```

Comment:
As we use fees for `send`, `stake`, `trade`, `wallet-connect` and we have `state.wallet.fees`. I would move this there.

<img width="362" alt="Screenshot 2025-06-15 at 10 37 00" src="https://github.com/user-attachments/assets/9d31cb74-8df0-4fde-9e17-e90d3dea1e31" />

We can have isFeeLoading for each network, last updated time, error (using default),... params there